app/models/laa_reference.rb

```diff
@@ -1,7 +1,25 @@
 # frozen_string_literal: true
 
 class LaaReference < ApplicationRecord
+  self.ignored_columns = %w[dummy_maat_reference]
```

Comment:
First step of two in dropping the `dummy_maat_reference` column

https://api.rubyonrails.org/classes/ActiveRecord/ModelSchema/ClassMethods.html#method-i-ignored_columns-3D